lib/index.js

```diff
@@ -22,11 +22,24 @@ var AppsFlyer = module.exports = integration('AppsFlyer')
  */
 
 AppsFlyer.ensure(function(msg, settings) {
-  if (contains(msg.library(), 'ios') && !settings.appleAppID) {
+
+  if (!msg.device().type) {
```

Comment:
since we are using `msg.device()` couple times in this function, we ought to make this a variable :)